src/main/java/org/testng/reporters/XMLStringBuffer.java

```diff
@@ -367,7 +358,6 @@ public void toWriter(Writer fw) {
   }
 }
 
-
 ////////////////////////
 
 class Tag {
```

Comment:
The class should be static

Reply:
This class is not a nested class within `XMLStringBuffer` and so we cannot add `static`

Reply:
ok :+1: